multiraft/multiraft_test.go

```diff
@@ -28,23 +28,25 @@ import (
 	"github.com/coreos/etcd/raft/raftpb"
 )
 
-var rand = util.NewPseudoRand()
+var testRand = util.NewPseudoRand()
 
 func makeCommandID() string {
-	return util.RandString(rand, commandIDLen)
+	return util.RandString(testRand, commandIDLen)
 }
 
 type testCluster struct {
-	t        *testing.T
-	nodes    []*state
-	tickers  []*manualTicker
-	events   []*eventDemux
-	storages []*BlockableStorage
+	t         *testing.T
+	nodes     []*state
+	tickers   []*manualTicker
+	events    []*eventDemux
+	storages  []*BlockableStorage
+	transport *localRPCTransport
 }
 
 func newTestCluster(size int, t *testing.T) *testCluster {
 	transport := NewLocalRPCTransport()
 	cluster := &testCluster{t: t}
+	cluster.transport = transport.(*localRPCTransport)
```

Comment:
newTestCluster should take a Transport (possibly nil) as input; then you wouldn't need a separate start() method. We could also move to the pattern used in storage/range_test.go before the number of inputs gets too unwieldy.